dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -97,14 +97,30 @@ def debug_str(self):
 
         return s
 
+    def check_import(self, lib_path):
+        import tensorflow as tf
+        import nvidia.dali as dali  # DALI symbols need to be loaded
```

Comment:
So we cannot install DALI and plugin in one go as DALI needs to be fully available here, right?